web/src/hooks/storage/api-model.ts

```diff
@@ -20,23 +20,14 @@
  * find current contact information at www.suse.com.
  */
 
-import { useMemo } from "react";
 import { useQuery, useSuspenseQuery } from "@tanstack/react-query";
 import { configModelQuery } from "~/queries/storage/config-model";
 import { apiModel } from "~/api/storage/types";
 import { QueryHookOptions } from "~/types/queries";
 
-function useApiModel(options?: QueryHookOptions): apiModel.Config | null {
+export default function useApiModel(options?: QueryHookOptions): apiModel.Config | null {
```

Comment:
No matter where default is placed. I still feeling is weird exporting hooks in that way

Reply:
I wrote `export default function` because  somehow I am following "this rule": if there is only one thing exported and it is exported by default (i.e., that cases in which the file name reveals what it is exporting) then I use `export default function` directly. Otherwise I export things at the end.